configuration/enable-offline-mode.adoc

```diff
@@ -46,14 +46,16 @@ include::modules/enable-offline-mode-roxctl.adoc[leveloffset=+2]
 [id="update-scanner-definitions-in-offline-mode"]
 == Updating Scanner definitions in offline mode
 
-Scanner contains a local vulnerability definitions database. When {product-title} runs in normal mode (connected to the internet), Scanner fetches new vulnerability definitions from the internet and updates its database.
+Scanner maintains a database of vulnerabilities that it retrieves from Central. When {rh-rhacs-first} is running in normal mode, Central retrieves the latest vulnerability data from the internet and passes it on to Scanner.
 //TODO: Add link to Vulnerability definitions
 
-However, when you are using {product-title} in offline mode, you must manually update Scanner definitions by uploading them to Central.
+However, if you are using {product-title-short} in offline mode, you must manually update the vulnerability data. To manually update the vulnerability data, you must upload a bundle to Central, which Scanner then retrieves.
```

Comment:
> ...you must upload a bundle to Central

nit: I don't believe we use `bundle` elsewhere, which may be OK given the context. Otherwise perhaps something along the lies of `definitions file` or `archive containing definitions`